taxi/views.py

```diff
@@ -79,6 +83,24 @@ class CarDeleteView(LoginRequiredMixin, generic.DeleteView):
     success_url = reverse_lazy("taxi:car-list")
 
 
+class CarDriverUpdateView(LoginRequiredMixin, generic.View):
+    @staticmethod
+    def post(
+        request: HttpRequest, pk: int
+    ) -> HttpResponseRedirect | HttpResponseBadRequest:
+        try:
+            car = Car.objects.get(pk=pk)
+        except Car.DoesNotExist:
+            return HttpResponseBadRequest("Car does not exist")
+        drivers = car.drivers
```

Comment:
Ensure that `drivers` is a related manager (e.g., a ManyToManyField) on the `Car` model. If it's not, accessing it directly like this will cause an error. Consider using `car.drivers.all()` or `car.drivers_set.all()` if it's a reverse relation.